client/src/constants/speed.ts

```diff
@@ -0,0 +1 @@
+export const defaultSpeed = 0.6;
```

Comment:
Consider using this constant in client/src/visualiser-src/linked-list-visualiser/initialiser.ts and deleting the one that's in visualiser-src/linked-list-visualiser/util/constants.ts. It's probably better to have one source of truth.

Reply:
Thanks Joanna, I've fixed that:)